build/server-error.js

```diff
@@ -8,15 +8,7 @@
 
 /* eslint-env node */
 
-const React = require('react');
-const RedBox = require('redbox-react').RedBoxError;
```

Comment:
If we're getting rid of this, we should remove the `redbox-react` dependency from package.json

Reply:
Yeah, good catch!